### ISSUE TITLE
Can't download ITV News after update 

### ISSUE THREAD

**Issue by m1ctk** [Ascoware/get-iplayer-automator#333]
I keep getting problem unknown all the time. It was fine before the latest update

**Comment by nitramlatep** [comment]
I think all content on ITV could be now blocked.  See the entry 331 which highlights content from ITV is not downloading.  It appears that ITV have DRM protected all their material.

**Comment by m1ctk** [comment]
Yes also now on the Kodi itv player add on all new progs since yesterday have had DRM applied but older shows still able to view though
